kolibri/core/assets/src/views/ExamReport/index.vue

```diff
@@ -148,28 +188,15 @@
         type: Number,
         required: true,
       },
+      tryIndex: {
```

Comment:
Could you please add brief docstrings to props that have something to do with attempts, tries, interactions, etc. like this one?

Reply:
Yes, can do - as the props have changed a lot here, I can add documentation to all of them for completeness.